### PR TITLE
Config file

### PR DESCRIPTION
**Pull request by Pagt0007**
Config file now has a reset function, changed addkey functionname to set
Now asks your oauth token if none is set
Also asks for room key if none is set

### CODE REVIEW

File: hypechat/__init__.py [hunk 1]
```diff
@@ -1,13 +1,26 @@
 import urllib2, json, urllib
 from configwriter import ConfigWriter
 
-token = ""
-config = ConfigWriter().get("meme")
+
+config = ConfigWriter()
+token = config.get("Oauth2token")
```
Comment (roelvanendhoven) [inline hypechat/__init__.py]: can be written more concise by using `if config.has("Oauth2token")

File: hypechat/__init__.py [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
+if token is None:
+    token = raw_input("OAut2htoken:")
+    config.set("Oauth2token", token)
+
+roomKey = config.get("roomkey")
```
Comment (roelvanendhoven) [inline hypechat/__init__.py]: Same goes for this one

File: hypechat/__init__.py [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
 
-token = ""
-config = ConfigWriter().get("meme")
+data = sys.argv
+if len(data) > 1:
+    if data[1] is not None:
+        ConfigWriter().reset()
+config = ConfigWriter()
+
+token = config.get("Oauth2token")
```
Comment (roelvanendhoven) [inline hypechat/__init__.py]: `if config.has("Oauth2token")`

**Comment by Pagt0007** [comment]
Changed those to config.has now

**Pull request merged**